subworkflows/local/db_check.nf

```diff
@@ -20,6 +21,22 @@ workflow DB_CHECK {
         .splitCsv ( header:true, sep:',' )
         .map { create_db_channels(it) }
 
+    // Download database for mOTUs
+    if( params.run_motus ){
+        check_motus_db =
+            parsed_samplesheet.filter{ it[0].tool == "motus" }
+                .ifEmpty{[]}
+        if( params.download_motus_db ){
+            MOTUS_DOWNLOADDB( params.motus_downloaddb_script )
+            check_motus_db = MOTUS_DOWNLOADDB.out.db
+                .map{[
+                    [tool: "motus", db_name: "db_mOTU", db_params: ''],
+                    it
+                ]}
+        }
```

Comment:
We want to remove that taxprofiler downloads the database for you.

you need to add to the motus CI test an additional commands that downloads and executes the download_db script itself, and then make a special database.csv that just species the path of the download db on the machine (i..e not a remote URL)

Reply:
No, I mean within the CI tests you have a command to download the  mOTUS download database python script, then run the script, then execute the pipeline

e.g.

```
run: |
      wget <url_to_download_script>
      wget <url_to_motu_specific_database_tsv>.tsv
      python <download_script>.py
      nextflow run nf-core/taxprofiler -profile docker,test_motus  --database <url_to_motu_specific_database_tsv>.tsv
```

Reply:
Great idea!